tests/test_data/tabs_and_spaces.robot

```diff
@@ -0,0 +1,21 @@
+*** Settings ***
+Library    OperatingSystem
```

Comment:
You can also use `Log  something    console=True` and drop this library import

Reply:
Actually the purpose was to have two lines in settings table, one separated with spaces and one with tabs. But I agree, I changed `Library` import to `Force Tags` to avoid unnecessary import.